packages/strapi-design-system/src/Box/BoxProps.js

```diff
@@ -25,22 +25,73 @@ BoxProps.defaultProps = {
 };
 
 BoxProps.propTypes = {
+  /**
+   * Javascript hover handler
```

Comment:
```suggestion
   * JavaScript hover handler
```

Why is the prop prefixed with an `_`?

Reply:
Because this prop should be hidden but I don't know how to hide props with Storybook ArgTable 😓

Reply:
You can't, but what you can do instead is to create a dummy component, define his props and use the dummy component's props in ArgsTable like we do for styled-components